src/modules/feature-modules/accessor/base/accessor-layout.component.spec.ts

```diff
@@ -16,9 +16,7 @@ import { AccessorLayoutComponent } from './accessor-layout.component';
 
 import { NotificationsService } from '@modules/shared/services/notifications.service';
 
-
```

Comment:
Why has the format of this file change so much, and with different approaches? With all these changes, is not easy to spot the differences...

- Why the comma at on the last properties of an object?
- When do you use "," or ";" to separate properties on a type?

We can change the patterns and ways to do things, but we should code all alike...

Thoughs?